### ISSUE TITLE
Updating dependencies from libgfortran3 to libgfortran5

### ISSUE THREAD

**Issue by tfoerst3r** [rcenvironment/rce#43]
During my installation process, I came across the following error:

~~~
error: Failed dependencies:
        libgfortran.so.3()(64bit) is needed by rce-10.3.0.202112090919-standard.x86_64
~~~

It seems that most modern Linux desktop systems do not support libgfortran3 anymore. Please consider updating the dependency.

**Comment by tfoerst3r** [comment]
`rpm --nodeps` worked but a new error occurred. 

> `Unexpected error when trying to acquire a file lock on /home/<home>/.rce/default/instance.lock`

Currently the domain https://software.dlr.de/ is not reachable. So no documentation can be downloaded. I am not sure how to install rce on openSUSE at this point.

**Comment by rmischke-dlr** [comment]
That `ProfileException` error is odd -- in the "normal" case that the profile directory was simply locked (because another instance was already using it), this should be caught and replaced with a more user-friendly message. Could you post the section from `/home/<home>/.rce/default/debug.log` that surrounds that error, or send us the whole zipped `debug.log` via email (the address on the website's starting page, above the news section)?

I suppose that you just started RCE without any special parameters? Did you launch it via GUI or from the command line?

The domain being down happened at the end of last week after a server reboot, and was fixed Monday morning. Sorry for the inconvenience.

**Comment by rmischke-dlr** [comment]
Please reopen this issue if you need additional feedback.

Regarding the main issue, as mentioned above, the fix will have to wait for RCE 11.0.0 for compatibility reasons.
